lock/src/main/java/org/infinispan/lock/impl/log/Log.java

```diff
@@ -13,7 +13,16 @@
 @MessageLogger(projectCode = "ISPN")
 public interface Log extends org.infinispan.util.logging.Log {
    String LOCK_DELETE_MSG = "The lock was deleted.";
+   String NODE_LEFT_MSG = "The node has left the cluster.";
+   String UNLOCK_FAILED_MSG = "LOCK[%s] Unlock failed from node %s";
```

Comment:
Why are these constants? Are you thinking about reusing them somewhere else?

Reply:
I used some in tests

Reply:
I used some of them on testing

Reply:
My bad, I was searching for references in the production scope in IntelliJ :)